src/components/ha-push-notifications-toggle.html

```diff
@@ -102,6 +102,7 @@
       );
   }
   unsubscribePushNotifications() {
+    if (!navigator.serviceWorker) return;
```

Comment:
Wouldn't it be better if we check for this in `pushSupported` and just hide the toggle?